credentials_windows.go

```diff
@@ -6,21 +6,56 @@ import (
 	"errors"
 	"fmt"
 	"os"
-	"path"
+	"path/filepath"
+	"strings"
 )
 
-func sourceHelpString(basepath string, name string) string {
-	s := "#\n"
-	s += fmt.Sprintf("# Credentials written to %s/\n", basepath)
-	s += "#\n"
-	s += fmt.Sprintf("\"%v\"\n", path.Join(basepath, "docker.cmd"))
-	s += fmt.Sprintf("# Run the command above to set your docker environment")
-	return s
+func credentialsNextStepsString(clusterName string) string {
+	return fmt.Sprintf("To see how to connect to your cluster, run: carina env %s --shell cmd|powershell|bash\n", clusterName)
+}
+
+func getCredentialFilePath(basepath string, shell string) string {
+	switch shell {
+	case "powershell":
+		return filepath.Join(basepath, "docker.ps1")
+	case "cmd":
+		return filepath.Join(basepath, "docker.cmd")
+	default: // Windows Bash
+		return filepath.Join(basepath, "docker.env")
+	}
+}
+
+func forceUnixPath(winPath string) string {
+	// Convert C:/ --> /C/
+	unixPath := "/" + strings.Replace(winPath, ":\\", "/", 1)
+	// Replace path seperators
+	unixPath = strings.Replace(unixPath, "\\", "/", -1)
+	return unixPath
```

Comment:
Yeah... it's a hack. :smile: I didn't see a nicer way of supporting bash on windows, which uses UNIX paths.